docs/src/main/paradox/journal.md

```diff
@@ -8,6 +8,29 @@
 - Messages written by a single persistent actor are partitioned across the cluster to achieve scalability with data volume by adding nodes.
 - @extref:[Persistence Query](akka:scala/persistence-query.html) support by `CassandraReadJournal`
 
+### Schema 
+
+The keyspace and tables needs to be created before using the plugin. Auto creation of the keyspace and tables
+is included as a development convenience and should never be used in production. Cassandra does not handle
+concurrent schema migrations well and if every Akka node tries to create the schema at the same time you'll
+get column id mismatch errors in Cassandra.
+
+The default keyspace used by the plugin is `akka`, it should be created with the
+NetworkTopology replication strategy with a replication factor of at least 3:
+
+```
+CREATE KEYSPACE IF NOT EXISTS akka WITH replication = {'class': 'NetworkTopologyStrategy', '<your_dc_name>' : 3 }; 
+```
+
+For local testing, and the default if you enable `cassandra-plugin.journal.keyspace-autocreate` you can use the following:
```

Comment:
This sentence doesn't parse for me, shouldn't it be "For local testing and the default ... _is_:"?